tests/conftest.py

```
@@ -89,7 +90,27 @@ def tests_tmp_dir(request: FixtureRequest, tmp_path_factory: TempPathFactory) ->
 
 
 @pytest.fixture(scope="session")
-def current_client_token(admin_client: DynamicClient) -> str:
+def current_client_token(
+    is_byoidc: bool, cluster_authentication: Authentication | None, admin_client: DynamicClient
+) -> str:
+    if is_byoidc:
+        url = cluster_authentication.instance.spec.oidcProviders[0].issuer.issuerURL
+        response = requests.post(
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)